core-services/prow/02_config/_plugins.yaml

```diff
@@ -565,6 +565,17 @@ config_updater:
         ci/api-build01-ci-devcluster-openshift-com:6443:
         - ci
       name: ci-operator-misc-configs
+    ci-operator/config/**/*v*.yaml:
```

Comment:
This is going to break a lot of stuff. Why you need that?

Reply:
my bad, i have replaced it with `ci-operator/config/**/tektoncd-pipeline-operator/*v*.yaml`